### PR TITLE
Feature/us4953 ng1 current series streaming

### PR DESCRIPTION
**Pull request by nogorilla**
US4953 implements display of current series content from SS CMS API

### CODE REVIEW

File: crossroads.net/app/live_stream/current_series/currentSeries.controller.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    this.tags         = [];
+    this.visible      = false;
+    this.response;
+    this.cmsService = CMSService;
+    this.modal = $modal;
+  }
+
+  $onInit() {
+    this.cmsService.getCurrentSeries().then((response) => {
+      console.log(response);
```
Comment (sseissiger) [inline crossroads.net/app/live_stream/current_series/currentSeries.controller.js]: Please delete log from production code.

File: crossroads.net/app/live_stream/current_series/currentSeries.controller.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      controllerAs: 'modal',
+      openedClass: 'crds-modal',
+      resolve: {
+        options: function() {
+          return options;
+        }
+      }
+    })
+    instance.rendered.then(() => {
+      document.getElementById('youtube-player').src = this.embed;
```
Comment (nogorilla) [inline crossroads.net/app/live_stream/current_series/currentSeries.controller.js]: not sure what you mean.

File: crossroads.net/app/live_stream/current_series_modal/currentSeriesModal.controller.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+export default class CurrentSeriesModalController {
+  constructor($modalInstance, options) {
+    this.modalInstance = $modalInstance;
+    console.log(options);
```
Comment (sseissiger) [inline crossroads.net/app/live_stream/current_series_modal/currentSeriesModal.controller.js]: Please remove logging.

File: crossroads.net/app/live_stream/current_series/currentSeries.controller.js (hunk 2 above)
Comment (sseissiger) [inline crossroads.net/app/live_stream/current_series/currentSeries.controller.js]: Hey, this is actually Victor on Sarah's github account. I am referring to not using DOM manipulation in Angular directly outside of directives. Not a big deal, feel free to ignore.

**Pull request merged**